test/stablecoin_converter.js

```diff
@@ -223,6 +223,186 @@ contract("StablecoinConverter", async (accounts) => {
       assert.equal((orderResult2.sellAmount).toNumber(), 10, "sellAmount was stored incorrectly")
       assert.equal((orderResult2.buyAmount).toNumber(), 5, "buyAmount was stored incorrectly")
     })
+    it("places two orders and first matches them partically and then fully in a 2nd solution submission", async () => {
```

Comment:
partically -> partially

Reply:
nit: maybe rephrase into "Accepts a second solution that achieves more volume than existing one"